app/javascript/controllers/course_description_card_controller.js

```diff
@@ -1,32 +1,35 @@
-import { Controller } from "@hotwired/stimulus"
+import { Controller } from "@hotwired/stimulus";
 
 export default class extends Controller {
+  static targets = [
+    "showMoreToggle",
+    "upArrow",
+    "downArrow",
+    "courseDescription",
+  ];
+  connect() {
+    console.log("Course card controller active");
+  }
```

Comment:
What is the change here ?

Reply:
Just removed a logger in showMoreToggle(event) {},but while saving it, the code hs been formatted